src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -1480,6 +1480,226 @@ public new abstract class Arm64 : AdvSimd.Arm64
         /// </summary>
         public static unsafe Vector<ulong> LoadVectorUInt32ZeroExtendToUInt64(Vector<ulong> mask, uint* address) => LoadVectorUInt32ZeroExtendToUInt64(mask, address);
 
+
+        ///  LoadVectorx2 : Load two-element tuples into two vectors
```

Comment:
As per other recent PRs, we're dropping the `LoadVectorx2` from this line (and similar for other APIs and the other file):
```
/// Load two-element tuples into two vectors
```